lib/driver.js

```diff
@@ -271,12 +271,26 @@ class AndroidDriver extends BaseDriver {
       if (this.opts.fastReset) {
         await helpers.resetApp(this.adb, this.opts.app, this.opts.appPackage, this.opts.fastReset);
       }
+      if (this.opts.autoGrantPermissions) {
+        try {
+          await this.adb.grantAllPermissions(this.opts.appPackage, this.opts.app);
```

Comment:
does this even make sense? we know in this case that `this.opts.app` is not defined. does it make sense to grant permissions in that case?